_posts/2023/2023-01-22-only-use-locals-in-view-partials.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Only use locals in view partials"
+description: "It helps keep your state manageable"
+layout: article
+category: ruby
+image:
+  base: '2023/only-use-locals-in-view-partials'
+  alt: "Local building sign"
+  credit: "Priscilla Du Preez"
+  source: "https://unsplash.com/photos/acNPOikiDRw"
+
+---
+
+The Rails view layer is incredibly flexible, but can easily become tricky to manage even though it’s the main way our users and customers access our work.
+
+https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials
+
+Many Ruby teams 
```

Comment:
Is this needed?

Maybe have an extra sentence beforehand explaining the two ways people may access variables?